hollow-explorer-ui/src/main/java/com/netflix/hollow/explorer/ui/pages/HollowExplorerPage.java

```diff
@@ -43,12 +43,18 @@ public HollowExplorerPage(HollowExplorerUI ui) {
     public void render(HttpServletRequest req, HttpServletResponse resp, HollowUISession session) throws IOException {
         VelocityContext ctx = new VelocityContext();
 
-        if(ui.getHeaderDisplayString() != null)
-            ctx.put("headerDisplayString", ui.getHeaderDisplayString());
-        
-        if(ui.getCurrentStateVersion() != Long.MIN_VALUE)
+        if (ui.getCurrentStateVersion() != Long.MIN_VALUE)
             ctx.put("stateVersion", ui.getCurrentStateVersion());
-        
+
+        String headerDisplayString = ui.getHeaderDisplayString();
+        ctx.put("headerDisplayString", headerDisplayString);
+
+        if (headerDisplayString != null) {
+            String headerDisplayURL = ui.getFromHeaderDisplayMap(headerDisplayString);
```

Comment:
might be good to null check or `containsKey` on this, otherwise we'll have a link that contains the word `null`

Reply:
thanks for pointing out this one. I was earlier adding a getOrDefault in the getter method itself with default as "" but decided to against it for not having a common default values for all entries in the map. Added null check here.